### ISSUE TITLE
Delete files and folders bug in CE

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/unifile#145]
> j'ai réussi à supprimer les fichiers que je souhaitais via l'interface GitHub, impossible de supprimer les fichiers via l'interface du cloud explorer.

Github service can not delete files or folders in a branch

**Comment by JbIPS** [comment]
Is that with the current version of Silex? I just tried on [prod](https://editor.silex.me):
* Connect to GitHub
* Create a branch
* Remove branch

No error, branch is no more (nor in CE, nor in GH).

I also tried the same with folder in a branch. Please confirm bug and provide some use case

**Comment by lexoyo** [comment]
Yes but I'm talking about files and folder in a branch, not branches

**Comment by lexoyo** [comment]
Here is what i do
![delete-folder](https://user-images.githubusercontent.com/715377/39173015-bd7342a8-47a3-11e8-97c9-1cc9babcc9b3.gif)

And the HTTP calls
![screenshot from 2018-04-24 09-41-24](https://user-images.githubusercontent.com/715377/39173034-c9bccc96-47a3-11e8-9eed-82e1860de0aa.png)
